templates/user/admin/overview.html.twig

```diff
@@ -26,57 +26,51 @@
 
   <hr>
 
-  <table class="table table-striped">
-    <thead>
-    <tr>
-      <th scope="col">{{ 'Username'|trans }}</th>
-      <th scope="col">{{ 'Roles'|trans }}</th>
-      <th scope="col">{{ 'Confirmed'|trans }}</th>
-      <th scope="col">{{ 'Enabled'|trans }}</th>
-      <th scope="col"></th>
-      <th scope="col"></th>
-    </tr>
-    </thead>
-    <tbody>
+  <div class="row g-3 mb-4">
     {% for user in users %}
-      <tr>
-        <th scope="row">{{ user.username }}</th>
-        <td>
-        {% for role in user.getDisplayRoles %}
-          {% if loop.index0 > 0 %},{% endif %}
-          {{ role|trans }}
-        {% endfor %}
-        </td>
-        <td>
-          {% if user.confirmed %}
-            <i class="text-success fas fa-check"></i>
-          {% else %}
-            <i class="text-danger fas fa-times"></i>
-          {% endif %}
-        </td>
-        <td>
-          {% if user.enabled %}
-            <i class="text-success fas fa-check"></i>
-          {% else %}
-            <i class="text-danger fas fa-times"></i>
-          {% endif %}
-        </td>
-        <td>
-          <a class="text-nowrap" href="{{ path('user_admin_edit', {user: user.id}) }}">
-            <i class="fas fa-edit" title="{{ 'Edit user'|trans }}" data-controller="tooltip" data-bs-placement="top"></i>
-          </a>
-        </td>
-        {% if is_granted('ROLE_ADMIN') %}
-          <td>
-            <a class="text-nowrap" href="{{ impersonation_path(user.username) }}" data-turbo-prefetch="false">
-              <i class="fas fa-sign-in-alt" title="{{ 'Switch to user'|trans }}" data-controller="tooltip" data-bs-placement="top"></i>
+      <div class="col-md-6 col-lg-4">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title mb-3">{{ user.username }}</h3>
+            <p class="mb-0">{{ 'Roles'|trans }}: {% for role in user.getDisplayRoles %}
```

Comment:
Zouden we de content van de p-tag niet op een nieuwe lijn zetten? Die if voor die `,` is ook maar raar, mss beter checken of het niet het laatste item is of niet? Dit lijkt mij leesbaarder:
```
<p>
  {{ 'Roles'|trans }}:
  {% for role in user.getDisplayRoles %}
    {{ role|trans }}
    {% if not loop.last %},{% endif %}
  {% endfor %}
</p>
```